test/e2e/ubernetes_lite.go

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"math"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/pkg/util/intstr"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+var _ = Describe("Ubernetes Lite", func() {
```

Comment:
No, this is actually core functionality, and one of the main features of our upcoming 1.2 release. 
The code on line 45 causes the tests to be automatically skipped on non-multi-zone clusters (which all our test clusters except one - "ubernetes-lite" - are, currently).
As soon as all our e2e tests have proved to be stable against multi-zone clusters (they are all currently passing as we speak), I intend to add the "ubernetes-lite" multi-zone test cluster to the list of merge-queue blocking test clusters.